### DISCUSSION TITLE
How do I know which log is associated with an op in the compute logs?

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#18030] [Q&A]
Hi, I've been searching for a way to reach the stdout &amp; stderr of ops for a certain job. I've read your documentation about Compute Log Storage:
<https://docs.dagster.io/deployment/dagster-instance#compute-log-storage>
And according to the default local behavior, Raw Compute Logs should reside in `$DAGSTER_HOME/storage/&lt;dagster_run_id&gt;/compute_logs/&lt;op/solid_name&gt;.out/err`
But when I go to this location on my machine, log files are not named according to op name, but some random hash that won't allow recognizing which log belongs to which op.
Is there a way of knowing which op each log belongs?

Thanks in advance

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C01U954MEER/p1700037592455129?thread_ts=1700037592.455129&cid=C01U954MEER).*

**Comment by alangenfeld** [comment]
> Is there a way of knowing which op each log belongs?

Not from the log file name alone. We changed the log capture set-up to be process based instead of specific op/asset in preparation for in process concurrent execution. 

The event log contains log captured events which contain the reference to the specific destination the logs were written out to, so technically you use these events to resolve the mapping. 


I've corrected the out of date docs here https://github.com/dagster-io/dagster/pull/18031
